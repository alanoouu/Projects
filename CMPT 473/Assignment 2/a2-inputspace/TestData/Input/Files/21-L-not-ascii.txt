header1;;-;;header2;;-;;header3.subfield3.subobject3;;-;;header3.subfield4;;-;;header3.subfield4.subobject5;;-;;header6.1;;-;;header6.2
"θ";;-;;"";;-;;;;-;;"νη";;-;;"φχ";;-;;"β";;-;;"ξ ζς";;-;;"μφθ";;-;;"λ "
"σω";;-;;"β""ψυ";;-;;"βχ";;-;;"οπ,ιμ";;-;;"";;-;;"";;-;;"ςχε"
"ιη";;-;;"βμ";;-;;"";;-;;"κα";;-;;" φφ δλ";;-;;"ιωχ";;-;;"ηκπτ"
"νς";;-;;"ρλ
ωο";;-;;"ηε";;-;;"φμ,τν";;-;;"ζ";;-;;"μς";;-;;"δοβ"
