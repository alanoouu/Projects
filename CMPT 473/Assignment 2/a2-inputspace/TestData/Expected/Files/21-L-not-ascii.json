[
  {
    "header1": "σω",
    "header2": "β\"ψυ",
    "header3": {
      "subfield3": {
        "subobject3": "βχ"
      },
      "subfield4": [
        {
          "subobject5": ""
        },
        "οπ,ιμ"
      ]
    },
    "header6": [
      "",
      "ςχε"
    ]
  },
  {
    "header1": "ιη",
    "header2": "βμ",
    "header3": {
      "subfield3": {
        "subobject3": ""
      },
      "subfield4": [
        {
          "subobject5": " φφ δλ"
        },
        "κα"
      ]
    },
    "header6": [
      "ιωχ",
      "ηκπτ"
    ]
  },
  {
    "header1": "νς",
    "header2": "ρλ\nωο",
    "header3": {
      "subfield3": {
        "subobject3": "ηε"
      },
      "subfield4": [
        "φμ,τν",
        {
          "subobject5": "ζ"
        }
      ]
    },
    "header6": [
      "μς",
      "δοβ"
    ]
  }
]